src/source/shift.rs

```diff
@@ -184,7 +184,7 @@ impl SourceParser for Shift {
             var parseScript = require('shift-parser').parseScript;
             var fs      = require('fs');
 
-            var source  = fs.readFileSync('{}', {{encoding: "utf-8"}});
+            var source  = fs.readFileSync({:?}, {{encoding: "utf-8"}});
```

Comment:
Why a debug print?

Reply:
To fix paths. Please see commit message for details :)

Reply:
Oops, lost track of the commit message :)

Could we use `String::escape_default` or `String::escape_unicode` here? I believe that this would be clearer.

Reply:
I guess, this just felt simpler to add because it also includes quotes and is shorter.

Reply:
Actually this will be immediately removed in the "reuse Node" PR so I guess not much point in adjusting specifics?

Reply:
Fair enough :)